public/app/features/dashboard-scene/serialization/transformSaveModelToScene.ts

```diff
@@ -292,21 +295,26 @@ export function createDashboardSceneFromDashboardModel(oldModel: DashboardModel,
         }
       : undefined;
 
-  const behaviorList: SceneObjectState['$behaviors'] = [
-    new behaviors.CursorSync({
-      sync: oldModel.graphTooltip,
-    }),
-    new behaviors.SceneQueryController({
+  const queryController = new behaviors.SceneQueryController(
+    {
       enableProfiling:
         config.dashboardPerformanceMetrics.findIndex((uid) => uid === '*' || uid === oldModel.uid) !== -1,
       onProfileComplete: getDashboardInteractionCallback(oldModel.uid, oldModel.title),
+    },
+    getDashboardSceneProfiler()
```

Comment:
@mdvictor - made the profiler a singleton. This way we can "continue" a profile session when reloadable dashboards are enabled. Previously, for each scene created a new profiler was instantiated.